packages/websocket-plugin/src/symbols.ts

```diff
@@ -74,6 +74,16 @@ export class DisconnectWebSocket {
   }
 }
 
+/**
+ * Action triggered when websocket is disconnected
+ */
+export class WebSocketDisconnected {
+  static get type() {
```

Comment:
static type = ''[Websocket] Disconnected';

Reply:
using existing pattern (see file), any reason why it should be this way?